python/postprocessing/modules/jme/jetmetHelperRun2.py

```diff
@@ -6,55 +6,65 @@
+                    '2018'   : 'Autumn18_V19_DATA'
                   }
 
-jecTagsDATA = { '2016B' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016C' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016D' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016E' : 'Summer16_07Aug2017EF_V11_DATA', 
-                '2016F' : 'Summer16_07Aug2017EF_V11_DATA', 
-                '2016G' : 'Summer16_07Aug2017GH_V11_DATA', 
-                '2016H' : 'Summer16_07Aug2017GH_V11_DATA', 
-                '2017B' : 'Fall17_17Nov2017B_V32_DATA', 
-                '2017C' : 'Fall17_17Nov2017C_V32_DATA', 
-                '2017D' : 'Fall17_17Nov2017DE_V32_DATA', 
-                '2017E' : 'Fall17_17Nov2017DE_V32_DATA', 
-                '2017F' : 'Fall17_17Nov2017F_V32_DATA', 
-                '2018A' : 'Autumn18_RunA_V19_DATA',
-                '2018B' : 'Autumn18_RunB_V19_DATA',
-                '2018C' : 'Autumn18_RunC_V19_DATA',
-                '2018D' : 'Autumn18_RunD_V19_DATA',
-                } 
-
-jerTagsMC = {'2016' : 'Summer16_25nsV1_MC',
-             '2017' : 'Fall17_V3_MC',
-             '2018' : 'Autumn18_V7_MC'
+jecTagsDATA = { '2016B'   : 'Summer16_07Aug2017BCD_V11_DATA', 
+                '2016C'   : 'Summer16_07Aug2017BCD_V11_DATA', 
+                '2016D'   : 'Summer16_07Aug2017BCD_V11_DATA', 
+                '2016E'   : 'Summer16_07Aug2017EF_V11_DATA', 
+                '2016F'   : 'Summer16_07Aug2017EF_V11_DATA', 
+                '2016G'   : 'Summer16_07Aug2017GH_V11_DATA', 
+                '2016H'   : 'Summer16_07Aug2017GH_V11_DATA', 
+                '2017B'   : 'Fall17_17Nov2017B_V32_DATA',
+                '2017C'   : 'Fall17_17Nov2017C_V32_DATA',
+                '2017D'   : 'Fall17_17Nov2017DE_V32_DATA',
+                '2017E'   : 'Fall17_17Nov2017DE_V32_DATA',
+                '2017F'   : 'Fall17_17Nov2017F_V32_DATA',
+                'UL2017B' : 'Summer19UL17_RunB_V5_DATA',
+                'UL2017C' : 'Summer19UL17_RunC_V5_DATA',
+                'UL2017D' : 'Summer19UL17_RunD_V5_DATA',
+                'UL2017E' : 'Summer19UL17_RunE_V5_DATA',
+                'UL2017F' : 'Summer19UL17_RunF_V5_DATA',
+                '2018A'   : 'Autumn18_RunA_V19_DATA',
+                '2018B'   : 'Autumn18_RunB_V19_DATA',
+                '2018C'   : 'Autumn18_RunC_V19_DATA',
+                '2018D'   : 'Autumn18_RunD_V19_DATA',
+              }
+
+jerTagsMC = { '2016'   : 'Summer16_25nsV1_MC',
+              '2017'   : 'Fall17_V3_MC',
+              'UL2017' : 'Summer19UL17_JRV2_MC',
+              '2018'   : 'Autumn18_V7_MC'
             }
 
 #jet mass resolution: https://twiki.cern.ch/twiki/bin/view/CMS/JetWtagging
 #nominal, up, down
-jmrValues = {'2016' : [1.0, 1.2, 0.8],
-             '2017' : [1.09, 1.14, 1.04],
-             '2018' : [1.24, 1.20, 1.28]
+jmrValues = { '2016'   : [1.0, 1.2, 0.8],
+              '2017'   : [1.09, 1.14, 1.04],
+              'UL2017' : [1.00, 1.00, 1.00], # placeholder
```

Comment:
note for later:
the jetMass scale and resolution use the placeholder values. basically no smearing/noscaling.
Given the large set of changes (taggerss algorithm, puppi tune) ... I think it's better to reset these for the UL and update later when proper values are checked as function of pt